.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,81 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
If you take a look at existing github workflows, you'll see the version references are done as a git hash instead of a version number like this. Can you convert this to do the same?

Reply:
Sure, I have replaced all version numbers by the related git hash.